packages/core/src/components/Dropdown/Dropdown.jsx

```diff
@@ -111,6 +120,10 @@ Dropdown.propTypes = {
    * Additional classes to be added to the root element.
    */
   className: PropTypes.string,
+  /**
+   * Optional prop for defining custom dropdown options. When using the `children` prop, `options` should be an empty list.
+   */
+  children: PropTypes.node,
```

Comment:
I'm not seeing the `children` prop in the doc site prop table, should I be?
I see `className` and then `defaultValue `but don't see `children`

Reply:
Tracked here: https://jira.cms.gov/browse/WNMGDS-349